.github/workflows/mergepr.yml

```
@@ -0,0 +1,32 @@
+name: Merge PR
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - master
+
+jobs:
+  merge-PR:
+    name: Deploy static Storybook
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    env:
+      working-dir: ./
+      TOKEN: ${{ secrets.GITHUB_ACCESS_TOKEN }}
```

This is unfortunately inconsistent across repos, making this a very easy mistake to make.

```suggestion
      TOKEN: ${{ secrets.GITHUBACCESSTOKEN }}
```